src/module-postfix.js

```diff
@@ -1,3 +1,8 @@
-module.exports.applyArabicShaping = applyArabicShaping;
-module.exports.processBidirectionalText = processBidirectionalText;
-module.exports.processStyledBidirectionalText = processStyledBidirectionalText;
+if (module) {
```

Comment:
This still fails if you would execute `index.js` in the browser. (`module is undefined`)
That's why in the prefix `module.exports` is passed as an alternative argument:
```js
(function (global, factory) {
  if (typeof exports === 'object' && typeof module !== 'undefined') {
    factory(module.exports)
  } else if (typeof define === 'function' && define.amd) {
    define(factory);
  } else {
    factory(global);
  }
}) (this, function (exports) {
```
so this check is actually performed there

Reply:
🤦‍♂️ Ah right of course. I tested the modified version by loading `index.js` in a browser and accessing `window.applyArabicShaping`, does the new code look right to you?